#! /usr/bin/python3

print("Traceback (most recent call last): ")
print("   File \"./move5.py\" line 30 in <module> ")
print("       canvas.move(rand_shape, 0, -120) ")
print("   File \"/usr/lib/python3.5/tkinter/__init__.py\", line 2443, in move ")
print("       self.tk.call((self._w, 'move') + args) ")
print("_tkinter.TclError: Invalid command name '.1990907152' ")
print("(this is a fake error)")
